src/views/Settings.vue

```diff
@@ -8,18 +8,15 @@
     <ion-content :fullscreen="true">
       <ion-item>
         <ion-icon :icon="businessOutline" slot="start" />
-        <ion-label>{{ $t("Store") }}</ion-label>
-      <ion-select interface="popover" :placeholder="$t('store name')">
-        <ion-select-option value="store1">Store 1</ion-select-option>
-        <ion-select-option value="store2">Store 2</ion-select-option>
-        <ion-select-option value="store3">Store 3</ion-select-option>
-        <ion-select-option value="store4">Store 4</ion-select-option>
+        <ion-label>{{ currentFacility.facilityId ? currentFacility.facilityId : '' }}</ion-label>
+      <ion-select interface="popover" :placeholder="$t('store name')" :selected-text="currentFacility.facilityId" @ionChange="setFacility($event)">
```

Comment:
We should set a value instead of selected text

Reply:
Used value prop on the place of selected -text prop